Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -112,12 +112,25 @@
             "visibleRule": "DeployToSlotFlag = true"
         },
         {
+            "name": "ImageSource",
+            "type": "pickList",
+            "label": "Image Source",
+            "defaultValue": "Registry",
+            "required": false,
+            "options": {
+                "Registry": "Container Registry",
+                "Builtin": "Built-in Image"
+            },
+            "helpMarkDown": "Select the option to choose the source of the image for your web-app.",
+            "visibleRule": "WebAppKind = applinux"
+        },
+        {
             "name": "DockerNamespace",
             "type": "string",
             "label": "Registry or Namespace",
             "defaultValue": "",
             "required": true,
-            "visibleRule": "WebAppKind = applinux",
+            "visibleRule": "ImageSource  = Registry",
```

Comment:
I think this should be and of the two conditions. Can you check switching to a windows app and see if the imageSource shows up?

Reply:
The imageSource doesn't show up when I switch to an windows app from a Linux webapp